### ISSUE TITLE
"something went wrong" when applying a filter to a post

### ISSUE THREAD

**Issue by AmesCornish** [glitch-soc/mastodon#2205]
### Steps to reproduce the problem

1. Click on the "..." in a post
2. Click on "filter this post"

### Expected behaviour

Options to filter the post

### Actual behaviour

I get "We're sorry, but something went wrong with the Mastodon app."

### Detailed description

Stack trace
-----------

```
Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7B%24%24typeof%2C%20type%2C%20compare%2C%20WrappedComponent%2C%20displayName%2C%20statusId%2C%20onChangeBackgroundColor%2C%20onClose%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7B%24%24typeof%2C%20type%2C%20compare%2C%20WrappedComponent%2C%20displayName%2C%20statusId%2C%20onChangeBackgroundColor%2C%20onClose%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at xr (https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:266091)
    at https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:270949
    at Bo (https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:280360)
    at ws (https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:318868)
    at mu (https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:311213)
    at _u (https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:311138)
    at su (https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:308168)
    at https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:259916
    at t.unstable_runWithPriority (https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:334798)
    at $i (https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:259625)
```Mapped stack trace
-----------

```
Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7B%24%24typeof%2C%20type%2C%20compare%2C%20WrappedComponent%2C%20displayName%2C%20statusId%2C%20onChangeBackgroundColor%2C%20onClose%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
Qg (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2664:35)
Qg (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2888:9)
R (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:3500:25)
L (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5447:24)
Qj (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5035:10)
Kj (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5029:26)
Ej (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:4815:6)
cg (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2331:17)
unstable_runWithPriority (webpack:///node_modules/scheduler/cjs/scheduler.production.min.js:252:11)
cg (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2304:9)
```


React component stack
---------------------

```

    in Unknown
    in c
    in Connect(c)
    in div
    in div
    in div
    in f
    in De
    in Connect(De)
    in div
    in div
    in t
    in t
    in kt
    in t
    in withRouter(kt)
    in InjectIntl(withRouter(kt))
    in Connect(InjectIntl(withRouter(kt)))
    in t
    in t
    in t
    in withRouter(t)
    in t
    in t
    in Rt
    in c
    in IntlProvider
    in Kt
```

### Specifications

Google Chrome 112.0.5615.121

**Comment by Plastikmensch** [comment]
This looks like #2186 which was fixed by bc7b3399fbab62ea096d10704ea2837c863a0a53

**Comment by ClearlyClaire** [comment]
Yeah it does sound like that indeed!
